funcx_web_service/routes/funcx.py

```diff
@@ -212,21 +221,41 @@ def submit(user: User):
         # this should raise a 500 because it prevented any tasks from launching
         return create_error_response(RequestKeyError(e), jsonify_response=True)
 
+    rc = g_redis_client()
+    task_group = None
+    if task_group_id and TaskGroup.exists(rc, task_group_id):
+        app.logger.debug(f'Task Group {task_group_id} submitted to by user {user_id} already exists, checking if user is authorized')
+        task_group = TaskGroup.from_id(rc, task_group_id)
```

Comment:
We should add a comment here that in the future we could get this from a cached function to speed this lookup.